src/models/spaces/populations/populations.ts

```diff
@@ -33,7 +33,15 @@ export const PopulationsModel = types
   .extend(self => {
 
     function updateIsPlaying() {
-      self.isPlaying = self.model.interactive.isPlaying;
+      if (self.isPlaying !== self.model.interactive.isPlaying) {
+        self.isPlaying = self.model.interactive.isPlaying;
+
+        if (self.isPlaying) {
+          // clear any subset information and return to viewing the live tail
+          const d: ChartDataModelType = self.model.chartData;
+          d.setDataSetSubset(-1, d.maxPoints);
+        }
+      }
```

Comment:
Couldn't this be handled by the chart observing its own `isPlaying` property? Since the chart is the one enforcing that a user can't scrub out of the live-tail view when `isPlaying` is true, it seems like this should behavior should be the chart's concern.

Reply:
Good idea - have moved this logic to the chart.